### PR TITLE
031 dice shake

### PR DESCRIPTION
**Pull request by FrancoisSalmon**

### CODE REVIEW

**Review by Gonaline — DISMISSED** [review]
ok for me

**Review by LoicBrassart — DISMISSED** [review]
Saw no Evil

File: frontend/src/components/Links/Play/style.jsx [hunk 1]
```diff
@@ -13,12 +13,12 @@ export default styled.div`
...
-  }
-  div:hover {
-    background-image: url("../src/assets/pictures/play-button-hover.svg");
-    background-size: contain;
-    width: 40vh;
-    height: 40vh;
+    &:hover {
+      background-image: url("../src/assets/pictures/play-button-hover.svg");
+      background-size: contain;
+      width: 40vh;
```
Comment (LuigiMenez) [inline frontend/src/components/Links/Play/style.jsx]: width in vh?

File: frontend/src/pages/Home/style.jsx [hunk 2]
```diff
@@ -6,12 +6,19 @@ export default styled.section`
...
   h1 {
     text-align: center;
     margin-bottom: 8vh;
+    font-family: var(--font-family-baseline);
+    font-weight: bold;
   }
   div {
-    width: 100%;
-    justify-content: center;
+    width: 40vh;
```
Comment (LuigiMenez) [inline frontend/src/pages/Home/style.jsx]: width in vh?

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
No vh for width, and trying to import the fonts into react

**Review by Gonaline — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me

**Review by LoicBrassart — COMMENTED** [review]
Saw no Evil

**Review by LoicBrassart — APPROVED** [review]
(Misclick, it's alright)

**Pull request merged**